VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimDetailsScreen.tsx

```diff
@@ -78,13 +78,15 @@ function ClaimDetailsScreen({ navigation, route }: ClaimDetailsScreenProps) {
   const { attributes } = claim || ({} as ClaimData)
   const { dateFiled } = attributes || ({} as ClaimAttributesData)
 
-  const [count, setCount] = useState(0)
+  const [count, setCount] = useState(-1)
```

Comment:
Just curious why this is changed from 0 to -1. Any potential side effects?

Reply:
No negative side effects. It was needed so that when count is actually 0 submit evidence is triggered or if greater than 0 file request is triggered